lib/langchain/llm/openai.rb

```diff
@@ -23,17 +23,12 @@ def initialize(api_key:, llm_options: {})
     # @param text [String] The text to generate an embedding for
     # @return [Array] The embedding
     #
-    def embed(text:)
-      model = DEFAULTS[:embeddings_model_name]
+    def embed(text:, **params)
+      parameters = {model: DEFAULTS[:embeddings_model_name], input: text}
```

Comment:
So you can't overwrite the model used in this `embed()` method?

Reply:
You can override by passing it in the params. On line `31` I'm doing a merge to these parameters.

Reply:
Oops -- didn't see it!